### PR TITLE
Current user

### PR DESCRIPTION
**Pull request by audreyli**
image

### CODE REVIEW

File: src/components/Fencing/FencingScoreboard.tsx [hunk 1]
```diff
@@ -31,8 +36,11 @@ export const FencingScoreboard: React.FC<{gameState: GameState; switchTeams(): v
...
           </tr>
-          {_.map(_.values(props.gameState.users), (user, i) => (
-            <tr key={i}>
+          {_.map(props.gameState.users, (user, userId) => (
+            <tr
+              key={userId}
+              className={userId === props.currentUserId ? 'fencing-scoreboard--current-user' : ''}
+            >
               <td>{user.displayName}</td>
               <td>{user.teamId}</td>
```
Comment (stevenhao) [inline src/components/Fencing/FencingScoreboard.tsx:45]: wdyt of moving the switch teams button into a thingy in here?

Comment (audreyli) [reply src/components/Fencing/FencingScoreboard.tsx:45]: o i like that

Comment (audreyli) [reply src/components/Fencing/FencingScoreboard.tsx:45]: image

any cute icon ideas

Comment (stevenhao) [reply src/components/Fencing/FencingScoreboard.tsx]: image
maybe this

**Pull request merged**